src/views/ProfessionalsPage.jsx

```diff
@@ -1,13 +1,25 @@
 // LIBRARY IMPORTS
-import React from 'react';
+import React, { useState, useRef } from 'react';
 
 // LOCAL IMPORTS
 import Card from '../components/Card.jsx';
+import SearchBar from '../components/SearchBar.jsx';
+import NoResults from '../components/NoResults.jsx';
+
+export function ProfessionalsPage({ data, setData, fullDataSet }) {
+  // SET STATES
+  const [clearedSearch, setClearedSearch] = useState(true);
 
-export function ProfessionalsPage() {
   return (
     <div>
-      <Card />
+      <h1 className='text-4xl font-bold text-center mt-12 text-primary'>You're in good company</h1>
+      <SearchBar
+        data={data}
+        setData={setData}
+        fullDataSet={fullDataSet}
+        setClearedSearch={setClearedSearch}
+      />
+      {clearedSearch ? <Card data={data} setData={setData} /> : <NoResults />}
```

Comment:
We're conditionally rendering the `Card` component or the `NoResults` component (the error message text block) based on the clearedSearch state. This took a lot of trial and error to figure out. At first I was using the states in `SearchBar` that determine what is seen on the form to also conditionally render here. However, that wasn't working and I realized I need a separate state to track if a search was successful or not.